dx/data-migration/README.md

```diff
@@ -25,6 +27,15 @@ This article is still under testing and development.
     # Copy something:
     $ mc cp -r prod/ckan/montreal-dev/uploads dx/dx-montreal-staging/
     ```
+6. Update the permission of `uploads` so that they are publically accessible and the images for groups/organizations can be loaded
+    ```
+    gsutil acl ch -r -u AllUsers:R gs://<bucket-name>/<file-location>/<file>
```

Comment:
How do you apply that for all the files recursively? I assume you didn't do that once for every different file.

Reply:
Updated the docs